app/controllers/application_controller.rb

```diff
@@ -133,7 +133,7 @@ def alert_and_redirect_moderated
       flash[:error] = I18n.t('application_controller.author_has_been_banned')
       redirect_to '/'
     elsif @node.status == 4 && (current_user && (current_user.role == 'admin' || current_user.role == 'moderator'))
-      flash[:warning] = "First-time poster <a href='#{@node.author.name}'>#{@node.author.name}</a> submitted this #{time_ago_in_words(@node.created_at)} ago and it has not yet been approved by a moderator. <a class='btn btn-default btn-sm' href='/moderate/publish/#{@node.id}'>Approve</a> <a class='btn btn-default btn-sm' href='/moderate/spam/#{@node.id}'>Spam</a>"
+      flash[:warning] = "First-time poster <a href='/profile/#{@node.author.name}'>#{@node.author.name}</a> submitted this #{time_ago_in_words(@node.created_at)} ago and it has not yet been approved by a moderator. <a class='btn btn-default btn-sm' href='/moderate/publish/#{@node.id}'>Approve</a> <a class='btn btn-default btn-sm' href='/moderate/spam/#{@node.id}'>Spam</a>"
```

Comment:
Nice work !!!

Reply:
please update the link here too  https://github.com/publiclab/plots2/blob/master/test/functional/notes_controller_test.rb#L290 .It will resolve the error.Thanks.

Reply:
done. please review its pull req at https://github.com/publiclab/plots2/pull/2482

Reply:
plz do the commit in this PR only

Reply:
done @Gauravano. Apologies for the inconvenience.

Reply:
No problem @yashbth